docs/user_guide/gui_workflow.rst

```diff
@@ -56,19 +59,21 @@ only after you know the API.
    #. Click once on the qubit.
    #. Click ``Shift+F2``
    #. Double-click on the qubit.
-   #. In the new *Object Properties* window locate a tab *PCell
+   #. In the new *Instance Properties* window locate a tab *PCell
       parameters*.
-   #. Scroll down in the list of parameters to locate *Coupler lengths*.
-   #. Change the value in the text box to ``100,160,0``.
+   #. Scroll down in the list of parameters to locate *Include coupler between qubits*.
+   #. Click on the box.
    #. Click ``Ok``.
 
 #. **Create a new qubit.**
 
    #. Press ``F2`` to see the whole top cell.
    #. From the *Tools* panel select *Instance*.
-   #. In the new *Object Editor Options* window locate *Library*.
-   #. Using the drop-down menu select *Element Library*.
-   #. Click on a button ``...`` a little bit to the left from the
+   #. Double-click on the qubit.
+   #. In the new *Instance Properties* window locate *Library*.
```

Comment:
```suggestion
   #. In the new *Editor Options* window locate *Library*.
```

This will be the name of the window if you click "Instance" from toolbar (instead of double-clicking existing qubit).